packages/plugin-loyalties-api/src/models/ScoreCampaigns.ts

```
@@ -262,7 +268,7 @@ export const loadScoreCampaignClass = (models: IModels, subdomain: string) => {
         }
       }
 
-      const changeScore = (eval(placeholder) || 0) * Number(currencyRatio) || 0;
+      let changeScore = (eval(placeholder) || 0) * Number(currencyRatio) || 0;
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)